cloudify_aws/common/decorators.py

```diff
@@ -479,6 +479,8 @@ def wrapper(**kwargs):
             ctx.node.properties.get('Tags'),
             ctx.instance.runtime_properties.get('Tags'),
             kwargs.get('Tags'))
+        if ctx.node.properties['cloudify_tagging'] == True:
```

Comment:
```suggestion
        if ctx.node.properties.get('cloudify_tagging', False):
```

When adding a property, I will usually use `.get`, because if someone is using  code that does not have this property, like from an old blueprint, then it will not fail from backward compat. Also, in Python, we don't generally need to do: `== False` or `== None` or `==True`. A boolean evaluation `if` is enough. That's the case unless you know that there is a chance that someone is mixing types, like if we change the type, or if the API doesn't send a boolean, but a string of a boolean value, i.e. `false` instead of `False`.